metrics/integration.go

```diff
@@ -115,6 +125,12 @@ func RegisterNewIntegrationPipelineRun(snapshotCreatedTime metav1.Time, pipeline
 	RegisterPipelineRunStarted(snapshotCreatedTime, pipelineRunStartTime)
 }
 
+// RegisterReleaseLatency, assess the time between HaveAppStudioTestsFinished and Release created. .
+func RegisterReleaseLatency(startTime metav1.Time) {
+	endCreateRelease := &metav1.Time{Time: time.Now()}
+	ReleaseLatencySeconds.Observe(endCreateRelease.Sub(startTime.Time).Seconds())
```

Comment:
You don't even need to do the `time.Now()` thing here.  You can just do 
```
latency := time.Since(startTime).Seconds()
ReleaseLatencySeconds.Observe(latency)
```